lib/features/mailbox_dashboard/presentation/model/search/search_email_filter.dart

```diff
@@ -70,6 +71,14 @@ class SearchEmailFilter with EquatableMixin {
     );
   }
 
+  T? _getOptionParam<T>(Option<T?>? option, T? defaultValue) {
```

Comment:
IMO, should make this function public so it can be reused. You can put it in a class in main. WDYT?

Reply:
it is very specific for this kind of function `copyWith` so I did not make it public

Reply:
What if I want to use it elsewhere in the future?